### PR TITLE
Customer story for Power Workplace

### PR DESCRIPTION
**Pull request by robmarcer**
Please note, the brief for customer stories when I conducted this interview was to focus on Node-RED rather than FlowFuse. This is the last customer story created under that brief.

### CODE REVIEW

**Comment by Yndira-E** [comment]
Let me update the image with a svg file instead

File: src/customer-stories/node-red-building-management.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+![Using FlowFuse and Node-RED to automate workplaces](./images/stories/power-workplace-diagram.svg "Using FlowFuse and Node-RED to automate workplaces"){data-zoomable}
+
+**Rob Marcer:** What were you using before Node-RED?
+
+**Alan Yeung:** We’ve used Node-RED almost since the start. Our background is systems integration and Audio Visual installation. We’ve tried working with external development teams but we found that the time-to-value for our development was too slow. Working on Node-RED ourselves allows us to rapidly deliver value as we can spend time on ‘coding’ rather than specification, costing, QA and deployment of new features.
+
+**Rob Marcer:** Can you think of any weaknesses in Node-RED that come to mind for you? 
+
+**Alan Yeung:** Version control could be stronger. If we have multiple people working on a flow it can be hard to spot where a new bug was introduced. We also see situations where Node-RED instances have to be restarted to fix a problem or deploy a new stack. We need to find better ways to manage those admin tasks.
+
```
Comment (Hasmin-AC) [inline src/customer-stories/node-red-building-management.md:55]: Add how Flowfuse is solving these issues.

**Comment by robmarcer** [comment]
@ZJvandeWeg I have added some counterpoints to the comments the customer made about version control in Node-RED. @Hasmin-AC assuming you are now happy with the content this can be published?

File: src/customer-stories/node-red-building-management.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+**Rob Marcer** FlowFuse is working to address the challenges of version control, we've implemented our DevOps Pipelines feature which allows our users to build, test and deploy their Node-RED applications with the confidence that they can esaily roll back to a previous state in a few clicks of a button. I expect a lot more to be added to FlowFuse in this area over the coming year.
```
Comment (gdziuba) [inline src/customer-stories/node-red-building-management.md:56]:
```suggestion
**Rob Marcer** FlowFuse is working to address the challenges of version control, we've implemented our DevOps Pipelines feature which allows our users to build, test and deploy their Node-RED applications with the confidence that they can easily roll back to a previous state in a few clicks of a button. I expect a lot more to be added to FlowFuse in this area over the coming year.
```

**Review by gdziuba — APPROVED** [review]
Minor grammar recommended changes.

**Comment by ZJvandeWeg** [comment]
@Hasmin-AC When is this scheduled to go out, it's a good one for the BMS market

**Comment by Hasmin-AC** [comment]
Need to update the PDF with the additional content. Will get that done and can publish some time in the week of april 22.

**Comment by Hasmin-AC** [comment]
Customer story published @Yndira-E for Social media

**Pull request merged**